File: client/Client/Network/Messages/Session/DenyUserLoginMessage.cs
namespace LoadRunnerClient.Messages
{
    public class DenyUserLoginMessage
    {
        public const string TYPE = "DenyUserLoginMessage";

        public DenyUserLoginMessage()
        {
        }

        public DenyUserLoginMessage(string reason)
        {
            _reason = reason;
        }

        private string _reason;

        public string reason
        {
            get => _reason;
            set => _reason = value;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/user/DenyUserLoginMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.user;

public class DenyUserLoginMessage {

	public static final String TYPE = "DenyUserLoginMessage";

	private String reason;

	public DenyUserLoginMessage() {
	}

	public DenyUserLoginMessage(String reason) {
		this.reason = reason;
	}

	public String getReason() {
		return this.reason;
	}

	public void setReason(String reason) {
		this.reason = reason;
	}
}
